packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -91,7 +91,8 @@ module.exports = {
         '**/protractor.conf.js', // protractor config
         '**/protractor.conf.*.js', // protractor config
         '**/karma.conf.js', // karma config
-        '**/.eslintrc.js' // eslint config
+        '**/.eslintrc.js', // eslint config
+        '**/eslint.config.{js,mjs,cjs}' // eslint flat config
```

Comment:
I won’t believe eslint allows any extension but .js here.

Reply:
@ljharb True. They mention the other extensions might be needed in different circumstances, but only the .js extension is automatically picked up. 

Updated now.